src/import_edf.jl

```diff
@@ -204,22 +208,49 @@ can be either a `String` giving the generated channel name, or a `Pair` mapping
 a canonical name to a list of alternatives that it should be substituted for
 (e.g., `"canonical_name" => ["alt1", "alt2", ...]`).
 
+`unit_alternatives` lists standardized unit names and alternatives that map to them.
+See `OndaEDF.STANDAR_UNITS` for defaults.
+
+`preprocess_labels(label::String)` is applied to raw edf signal header labels
+beforehand; defaults to `identity`.
+
 See `OndaEDF.STANDARD_LABELS` for the labels (`signal_names => channel_names`
 `Pair`s) that are used to extract EDF signals by default.
 
 """
-function extract_channels_by_label(edf::EDF.File, signal_names, channel_names)
+function extract_channels_by_label(edf::EDF.File, signal_names, channel_names; unit_alternatives=STANDARD_UNITS, preprocess_labels=identity)
     matcher = x -> begin
         # yo I heard you like closures
-        return s -> match_edf_label(s.header.label, signal_names,
-                                    x isa Pair ? first(x) : x,
-                                    channel_names)
+        return s -> begin
+            m = match_edf_label(preprocess_labels(s.header.label),
+                                signal_names,
+                                x isa Pair ? first(x) : x,
```

Comment:
Now that this is in two places, could you lift this logic out with a comment explaining what `x` is, why it might be a pair, and why, if it is, `match_edf_label` needs the `first` of it?